### ISSUE TITLE
Can biopert just predict the realtion using pertained models and not train as given in Biobert formula

### ISSUE THREAD

**Issue by dhwani2410** [dmis-lab/biobert#99]
I have two questions related to this:-

python run_re.py --task_name=$TASK_NAME **--do_train=true** --do_eval=true --do_predict=true --vocab_file=$BIOBERT_DIR/vocab.txt --bert_config_file=$BIOBERT_DIR/bert_config.json --init_checkpoint=$BIOBERT_DIR/model.ckpt-1000000 --max_seq_length=128 --**train_batch_size=32** --learning_rate=2e-5 --num_train_epochs=3.0 --do_lower_case=false --data_dir=$RE_DIR --output_dir=$OUTPUT_DIR

1) **Can we run the prediction and not train**. For the same how should the input file be made? As the input file to train consist of three columns 1)Index 2)Sentences with (NER replaced with a keyword) 3) Labels. 
Since I want to run in an unknown dataset and I don't have labels, can just **Sentence** column be the input to model?

2) **What kind of a relation does the model tell**? Is it positive or negative or just that there exists some relationship between the NER present in the sentences(for example GENES and DISEASES)

**Comment by wangxinyi-gsafety** [comment]
@dhwani2410, I have same question with you, have you got some answer regarding this?
